hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -4566,6 +4567,7 @@
           width: 1
           default: lc_ctrl_pkg::Off
           inst_name: csrng
+          top_signame: lc_ctrl_lc_hw_debug_en
```

Comment:
There is talk of removing this feature still, once it is verified as working first. @msfschaffner should we leave this off for now, or just remove it later if that is the decision?

Reply:
OK I see. When do you think will that decision be made?

If this does not affect the top-level functionality / break simulation, I am in favor of connecting it and removing later in order to account for the connections.